### DISCUSSION TITLE
Exclude content indexing for certain file types

### DISCUSSION THREAD

**Discussion by aaltonenp** [dadoonet/fscrawler#1308] [Q&A]
Is it possible to exclude certain file types from content indexing but still index file system metadata in a single crawler job? Or do I need to create two jobs?

For example, I don't want to index content of DWG files because I get a lot of "Unsupported AutoCAD drawing version" errors and I don't need the content. But I still want the metadata for these files.

Do I just exclude all .dwg files on the first crawler job and then create a second that only includes those and doesn't index content? Can the jobs run in parallel or do I need to set them to run back-to-back?

**Comment by dadoonet** [comment]
It's not possible ATM and indeed would require to have 2 jobs.

i think that could be a nice feature to have. 

Make the index content feature a bit smarter than true/false...

would you like to open a feature request?

**Reply by aaltonenp** [reply]
Sure, let's make it a feature request.
